Bottom: e546223bcf115ba77751ff9916d4246d749ada8d
Top:    e5390a41cf2c26cf7210e2988c678cde883d184f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-07 01:02:22 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index d7b5e34..ae0b9fa 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
